packages/localize/src/tools/test/extract/integration/main_spec.ts

```diff
@@ -0,0 +1,226 @@
+        `  <msg id="8669027859022295761"><source>test_files/test.js:2</source>try<ph name="PH"/>me</msg>`,
+        `</messagebundle>\n`,
+      ].join('\n'));
+    });
+
+    it('should extract translations from source code, and write as XLIFF 1.2 format', () => {
+      extractTranslations({
+        rootPath,
+        sourceLocale: 'en-CA',
+        sourceFilePaths: [sourceFilePath],
+        format: 'xliff',
+        outputPath,
+        logger,
+        useSourceMaps: false,
+        useLegacyIds: false,
+      });
+      expect(fs.readFile(outputPath)).toEqual([
+        `<?xml version="1.0" encoding="UTF-8" ?>`,
+        `<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">`,
+        `  <file source-language="en-CA" datatype="plaintext">`,
+        `    <body>`,
+        `      <trans-unit id="3291030485717846467" datatype="html">`,
+        `        <source>Hello, <x id="PH"/>!</source>`,
+        `        <context-group purpose="location">`,
+        `          <context context-type="sourcefile">test_files/test.js</context>`,
+        `          <context context-type="linenumber">2</context>`,
+        `        </context-group>`,
+        `      </trans-unit>`,
+        `      <trans-unit id="8669027859022295761" datatype="html">`,
+        `        <source>try<x id="PH"/>me</source>`,
+        `        <context-group purpose="location">`,
+        `          <context context-type="sourcefile">test_files/test.js</context>`,
+        `          <context context-type="linenumber">3</context>`,
+        `        </context-group>`,
+        `      </trans-unit>`,
+        `    </body>`,
+        `  </file>`,
+        `</xliff>\n`,
+      ].join('\n'));
+    });
+
+    it('should extract translations from source code, and write as XLIFF 2 format', () => {
+      extractTranslations({
+        rootPath,
+        sourceLocale: 'en-AU',
+        sourceFilePaths: [sourceFilePath],
+        format: 'xliff2',
+        outputPath,
+        logger,
+        useSourceMaps: false,
+        useLegacyIds: false,
+      });
+      expect(fs.readFile(outputPath)).toEqual([
+        `<?xml version="1.0" encoding="UTF-8" ?>`,
+        `<xliff version="2.0" xmlns="urn:oasis:names:tc:xliff:document:2.0" srcLang="en-AU">`,
+        `  <file>`,
+        `    <unit id="3291030485717846467">`,
+        `      <segment>`,
+        `        <source>Hello, <ph id="1" equiv="PH"/>!</source>`,
+        `      </segment>`,
+        `    </unit>`,
+        `    <unit id="8669027859022295761">`,
+        `      <segment>`,
+        `        <source>try<ph id="1" equiv="PH"/>me</source>`,
+        `      </segment>`,
+        `    </unit>`,
+        `  </file>`,
+        `</xliff>\n`,
+      ].join('\n'));
+    });
+
+    for (const target of ['es2015', 'es5']) {
+      it('should use source-maps to find the original location of translations', () => {
```

Comment:
nit: may be include `target` into the test description:
```suggestion
      it(`should use source-maps to find the original location of translations (using ${target} bundle)`, () => {
```